electrolytes/solvation_shell_utils.py

```diff
@@ -0,0 +1,77 @@
+import copy
+import itertools
+import numpy as np
+
+
+def rmsd(a, b):
+    """
+    Compute the root mean squared distance between two sets of coordinates
+    Args:
+        a: numpy array of positions, shape [N_atoms, 3]
+        b: numpy array of positions, shape [N_atoms, 3]
+    """
+    return np.sqrt(np.mean(np.linalg.norm(b - a, axis=1)))
```

Comment:
Couple notes for myelf (@levineds , @espottesmith  feel free to weigh in): I think we should make this function translation invariant by subtracting off the center of mass of both `a` and `b` or something like that. Accounting for rotation is going to be trickier; we could represent coordinates as pairwise displacements instead of absolute coordinates. I think PBCs also need to be explicitly accounted for in this function by wrapping the coordinates. We currently do this after the RMSD computation (right before dumping the files) but it should be done here.

Reply:
The algorithm for rigid rotation/translation RMSD is a solved problem. After adjusting for PBC, we can just use that. Example is here: https://github.com/charnley/rmsd